### PR TITLE
Add missing features for Window API

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR is a part of a project to add missing interfaces and interface features to BCD that are from an active spec (including WICG specs) and is supported in at least one browser.  This particular PR adds missing features, populating the results using data from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v1.1.7), for the `Window` API.

Spec: https://html.spec.whatwg.org/multipage/window-object.html#window, https://html.spec.whatwg.org/multipage/obsolete.html#Window-partial, https://wicg.github.io/cookie-store/, https://drafts.csswg.org/css-nav-1/, https://drafts.csswg.org/css-nav-1/, https://drafts.csswg.org/cssom-view/, https://w3c.github.io/webappsec-trusted-types/dist/spec/

IDL: https://github.com/w3c/webref/blob/master/ed/idl/html.idl

Tests Used: https://mdn-bcd-collector.appspot.com/tests/api/Window

### CODE REVIEW

File: api/Window.json [hunk 1]
```diff
@@ -4418,6 +4559,55 @@
           }
         }
       },
+      "navigate": {
```
Comment (foolip) [inline api/Window.json:4562]: I don't think we should add this entry, as it's not in any recent browser release.

**Review by foolip — CHANGES_REQUESTED** [review]

**Comment by foolip** [comment]
@vinyldarkscratch can you check if any of these entries also existing somewhere else in BCD, in particular on WindowOrWorkerGlobalScope?

**Comment by vinyldarkscratch** [comment]
Before submitting this PR, I had double-checked to make sure that the only entries being added were directly implemented on the `Window` interface!

**Comment by foolip** [comment]
Okay, in that case let me review properly...

**Review by foolip — APPROVED** [review]

**Pull request merged**